src/test/java/seedu/address/model/person/driver/DriverTest.java

```diff
@@ -33,12 +33,8 @@ public void isSameDriver() {
         // null -> returns false
         assertFalse(DRIVER_ALICE.isSameDriver(null));
 
-        // same name, all other attributes different -> returns true
```

Comment:
should an additional test case be added for if driver A and B has the same phone number but different names?